apps/studio/components/interfaces/Auth/AdvancedAuthSettingsForm/AdvancedAuthSettingsForm.tsx

```diff
@@ -190,6 +230,28 @@ const AdvancedAuthSettingsForm = () => {
                 header={<FormSectionLabel>Multi-Factor Authentication (MFA)</FormSectionLabel>}
               >
                 <FormSectionContent loading={isLoading}>
+                  <FormField
+                    name="MFA_TOTP"
+                    properties={{
+                      type: 'select',
+                      title: 'TOTP',
+                      description: 'Control Use Of TOTP Factors',
```

Comment:
Use "App Authenticator" instead of TOTP (or put it in parenthesis).

I also think the text style should be sentence case.